Mathlib/Order/SuccPred/InitialSeg.lean

```diff
@@ -48,6 +48,9 @@ theorem isSuccPrelimit_apply_iff (f : α ≤i β) : IsSuccPrelimit (f a) ↔ IsS
 theorem isSuccLimit_apply_iff (f : α ≤i β) : IsSuccLimit (f a) ↔ IsSuccLimit a := by
   simp [IsSuccLimit]
 
+alias ⟨_, map_isSuccPrelimit⟩ := isSuccPrelimit_apply_iff
```

Comment:
These ones can have dot-notation, right? IsSuccLimit.map, maybe?

Reply:
The full name here is actually `InitialSeg.map_isSuccPrelimit`, so it already uses dot notation!